runtime/CMakeLists.txt

```diff
@@ -150,6 +150,13 @@ else()
 	add_custom_target(j9vm_nlsgen)
 endif()
 
+# Generate jvmti.h
+add_custom_command(OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/../include/jvmti.h"
```

Comment:
The `/../` portion feels wrong - did you try without it?

Reply:
Yes, the ${CMAKE_CURRENT_BINARY_DIR}
If we build without it, it will generate file in `build/linux...../vm/build/include` which is not correct. We should build in `build/linux..../vm/include`

Reply:
Perhaps you want to use `${CMAKE_CURRENT_SOURCE_DIR}` instead so you don't make assumptions about the relationship between `CMAKE_CURRENT_SOURCE_DIR` and `CMAKE_CURRENT_BINARY_DIR `.